_posts/2021-10-18-amiti-uttarwar-p2p.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Amiti Uttarwar and the P2P network"
+tags:
+  - p2p
+  - eclipse attacks
+  - addrman
+---
+
+Amiti returns to the Chaincode office and to discuss all things p2p.
```

Comment:
should remove "and" 
"Amiti returns to the Chaincode office to discuss all things p2p."